mtt/config/Run3/config_2022.py

```diff
@@ -0,0 +1,1508 @@
+                "2022preEE": 0.858, "2022postEE": 0.866,
+            }[tag_key],
+            "very tight": {
+                "2022preEE": 0.979, "2022postEE": 0.980,
+            }[tag_key],
+        },
+    }
+
+    #
+    # selector configuration
+    #
+
+    # lepton selection parameters
+    cfg.x.lepton_selection = DotDict.wrap({
+        "mu": {
+            "column": "Muon",
+            "min_pt": {
+                "low_pt": 30,
+                "high_pt": 55,
+            },
+            "max_abseta": 2.4,
+            "iso": {
+                "column": "pfIsoId",
+                "value": 4,
+            },
+            "id": {
+                "low_pt": {
+                    "column": "tightId",
+                    "value": True,
+                },
+                "high_pt": {
+                    "column": "highPtId",
+                    "value": 2,
+                },
+            },
+            # veto events with additional leptons passing looser cuts
+            "min_pt_addveto": 25,
+            "id_addveto": {
+                "column": "tightId",
+                "value": True,
+            },
+            "max_abseta_addveto": 2.4,
+        },
+        "e": {
+            "column": "Electron",
+            "min_pt": {
+                "low_pt": 35,
+                "high_pt": 120,
+            },
+            "max_abseta": 2.5,
+            "barrel_veto": [1.44, 1.57],
+            "mva_id": {
+                "low_pt": "mvaIso_WP80",
+                "high_pt": "mvaNoIso_WP80",
+            },
+            # veto events with additional leptons passing looser cuts
+            "min_pt_addveto": 25,
+            "id_addveto": {
+                "column": "cutBased",
+                "value": 3,
+            },
+            "max_abseta_addveto": 2.5,
+        },
+    })
+
+    # jet selection parameters
+    cfg.x.jet_selection = DotDict.wrap({
+        "ak4": {
+            "column": "Jet",
+            "max_abseta": 2.5,
+            "min_pt": {
+                "baseline": 30,
+                "e": [50, 40],
+                "mu": [50, 50],
+            },
+            "btagger": {
+                "column": "btagDeepFlavB",
+                "wp": cfg.x.btag_wp["deepjet"]["medium"],
+            },
+        },
+        "ak8": {
+            "column": "FatJet",
+            "max_abseta": 2.5,
+            "min_pt": 400,
+            "msd": [105, 210],
+            "toptagger": {
+                "column": "particleNetWithMass_TvsQCD",
+                "wp": cfg.x.toptag_wp["particle_net"]["tight"],
+            },
+            "delta_r_lep": 0.8,
+        },
+    })
+
+    # MET selection parameters
+    cfg.x.met_selection = DotDict.wrap({
+        "column": "MET",
+        "min_pt": {
+            "e": 60,
+            "mu": 70,
+        },
+    })
+
+    # lepton jet 2D isolation parameters
+    cfg.x.lepton_jet_iso = DotDict.wrap({
+        "min_pt": 15,
+        "delta_r": 0.4,
+        "pt_rel": 25,
```

Comment:
Suggest to add `min_` to param names to make the cuts more explicit.
```suggestion
        "min_delta_r": 0.4,
        "min_pt_rel": 25,
```